src/app/components/client/toolbar/AnnouncementDialog.tsx

```diff
@@ -0,0 +1,404 @@
+                  <Image
+                    className={styles.bigImg}
+                    // The image rendering logic is skipped in coverage reports because
+                    // it relies on static imports of SVGs that are difficult to mock/test
+                    /* c8 ignore start */
+                    src={
+                      !bigImageUnavailableMap[
+                        relevantAnnouncement.announcement_id
+                      ]
+                        ? `/images/announcements/${relevantAnnouncement.announcement_id}/big.svg`
+                        : `/images/announcements/fallback/big.svg`
+                    }
+                    /* c8 ignore end */
+                    alt={l10n.getString("announcement-big-img-alt")}
+                    width={300}
+                    height={100}
+                    // The image rendering logic is skipped in coverage reports because
+                    // it relies on static imports of SVGs that are difficult to mock/test accurately in JSDOM
+                    /* c8 ignore start */
+                    onError={() =>
+                      setBigImageUnavailableMap((prev) => ({
+                        ...prev,
+                        [relevantAnnouncement.announcement_id]: true,
+                      }))
+                    }
+                    /* c8 ignore end */
+                  />
+                  <div className={styles.announcementWrapperOpen}>
+                    <dl className={styles.announcementItemOpen}>
+                      <dt>
+                        <LocalizedAnnouncementString
+                          announcement={relevantAnnouncement}
+                          type="title"
+                        />
+                      </dt>
+                      <dd>
+                        <LocalizedAnnouncementString
+                          announcement={relevantAnnouncement}
+                          type="description"
+                        />
+                      </dd>
+                    </dl>
+                    {relevantAnnouncement.cta_link && (
+                      <TelemetryLink
+                        href={relevantAnnouncement.cta_link}
+                        target="_blank"
+                        className={styles.announcementCta}
+                        eventData={{
+                          link_id: `${relevantAnnouncement.announcement_id}-cta`,
+                        }}
+                      >
+                        <LocalizedAnnouncementString
+                          announcement={relevantAnnouncement}
+                          type="cta-label"
+                        />
+                      </TelemetryLink>
+                    )}
+                  </div>
+                  <button
+                    className={styles.backBtn}
+                    onClick={() => setAnnouncementDetailsView(false)}
+                  >
+                    <span> {l10n.getString("announcement-dialog-back")}</span>
```

Comment:
```suggestion
                    <span>{l10n.getString("announcement-dialog-back")}</span>
```

… or maybe even (why is the `<span>` necessary?):

```suggestion
                    {l10n.getString("announcement-dialog-back")}
```